dags/multipod/mxla_maxtext_nightly_gke.py

```diff
@@ -0,0 +1,144 @@
+  ).run()
+
+  maxtext_nightly_2slice_v4_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V4,
+      tpu_cores=8,
+      num_slices=2,
+      tpu_zone=Zone.US_CENTRAL2_B.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+  ).run()
+
+  maxtext_nightly_4slice_v4_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V4,
+      tpu_cores=8,
+      num_slices=4,
+      tpu_zone=Zone.US_CENTRAL2_B.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+  ).run()
+
+  maxtext_nightly_8slice_v4_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V4,
+      tpu_cores=8,
+      num_slices=8,
+      tpu_zone=Zone.US_CENTRAL2_B.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+  ).run()
+
+  maxtext_nightly_1slice_v5p_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V5P,
+      tpu_cores=8,
+      cluster_name=ClusterName.V5P_8_MULTISLICE_CLUSTER.value,
+      tpu_zone=Zone.US_EAST5_A.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+      project_name=Project.CLOUD_TPU_MULTIPOD_DEV.value,
+  ).run()
+
+  maxtext_nightly_2slice_v5p_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V5P,
+      tpu_cores=8,
+      num_slices=2,
+      cluster_name=ClusterName.V5P_8_MULTISLICE_CLUSTER.value,
+      tpu_zone=Zone.US_EAST5_A.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+      project_name=Project.CLOUD_TPU_MULTIPOD_DEV.value,
+  ).run()
+
+  maxtext_nightly_4slice_v5p_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V5P,
+      tpu_cores=8,
+      num_slices=4,
+      cluster_name=ClusterName.V5P_8_MULTISLICE_CLUSTER.value,
+      tpu_zone=Zone.US_EAST5_A.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+      project_name=Project.CLOUD_TPU_MULTIPOD_DEV.value,
+  ).run()
+
+  maxtext_nightly_8slice_v5p_8 = gke_config.get_gke_maxtext_nightly_config(
+      tpu_version=TpuVersion.V5P,
+      tpu_cores=8,
+      num_slices=8,
+      cluster_name=ClusterName.V5P_8_MULTISLICE_CLUSTER.value,
+      tpu_zone=Zone.US_EAST5_A.value,
+      time_out_in_min=60,
+      test_name=default_test_name,
+      docker_image=jax_nightly_image.value,
+      test_owner=test_owner.TONY_C,
+      project_name=Project.CLOUD_TPU_MULTIPOD_DEV.value,
+  ).run()
+
+  (
+      maxtext_nightly_1slice_v4_8
```

Comment:
@RissyRan do you know if airflow has a utility to chain tasks in an iterable? I didn't turn anything up when I searched.